multitenancy/tenant-manager-datamodel/src/main/java/io/apicurio/multitenant/api/TenantsResource.java

```diff
@@ -36,18 +39,20 @@
   void updateTenant(@PathParam("tenantId") String tenantId, UpdateRegistryTenantRequest data);
 
   /**
-   * Deletes an existing `Tenant`.
+   * Marks an existing `Tenant` to be deleted.
    */
   @Path("/{tenantId}")
   @DELETE
   void deleteTenant(@PathParam("tenantId") String tenantId);
 
   /**
-   * Gets a list of all `Tenant` entities.
+   * Gets a list of `RegistryTenant` entities according to the query parameters set.
    */
   @GET
   @Produces("application/json")
-  List<RegistryTenant> getTenants();
+  RegistryTenantList getTenants(@QueryParam("status") String status,
+      @QueryParam("offset") Integer offset, @QueryParam("limit") Integer limit,
+      @QueryParam("order") SortOrder order, @QueryParam("orderby") SortBy orderby);
```

Comment:
Add validation to `offset` and `limit`, e.g. `@Max(1000)`. This should be generated from OpenAPI schema.

Reply:
Looks like this kind of things cannot be made with Studio

Reply:
done